packages/core/src/controllers/subscriber.ts

```diff
@@ -280,11 +237,14 @@ export class Subscriber extends ISubscriber {
   }
 
   private async setRelayerSubscriptions(subscriptions: SubscriberTypes.Active[]): Promise<void> {
-    await this.client.storage.setItem<SubscriberTypes.Active[]>(this.storageKey, subscriptions);
+    await this.relayer.core.storage.setItem<SubscriberTypes.Active[]>(
```

Comment:
`this.relayer.core.storage` This looks a bit unexpected, I'd expect it to be like `this.core.storage` ? i.e. relayer should come from core?